core/internal/mocks/service.go

```diff
@@ -0,0 +1,76 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+
+package mocks
```

Comment:
Put in its own package?

Reply:
Since rearranging the file structure for our mocks touches other systems too, I say we tackle this in a separate PR.